processing_r/processing/algorithm.py

```diff
@@ -291,16 +291,6 @@ def process_metadata_line(self, line):  # pylint: disable=too-many-return-statem
                                                                     self.tr('HTML files (*.html)'), optional=True))
             return
 
-        # dontuserasterpackage is the older version, should be considere obsolete
-        if line.lower().strip().startswith('load_raster_using_rgdal') or \
```

Comment:
Should we raise a QgsProcessingException with a nice explanatory message if this line is found?

Reply:
True. Fixed.